packages/now-cli/src/util/input/input-root-directory.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'path';
+import inquirer from 'inquirer';
+
+export async function inputRootDirectory(autoConfirm: boolean) {
+  if (autoConfirm) {
+    return null;
+  }
+
+  const { rootDirectory } = await inquirer.prompt({
+    type: 'input',
+    name: 'rootDirectory',
+    message: `In which directory is your code located?`,
+    transformer: (input: string) => {
+      return input ? input : '[.] ';
+    },
```

Comment:
Should we add a `default` here?

Reply:
I've had that at first, but when `default` is used it shows `(.)`, which doesn't look that good.

Reply:
Maybe `current`, or `./`. It's hard to understand that leaving the field empty will automatically choose the current directory.

Reply:
I've added the current basename, just dimmed